autode/hessians.py

```diff
@@ -318,43 +319,19 @@ def frequencies_proj(self) -> List[Frequency]:
 
         return trans_rot_freqs + vib_freqs
 
+    def copy(self) -> 'Hessian':
+        return self.__class__(np.copy(self), units=self.units, atoms=self.atoms)
 
-def calculate_numerical_hessian(species, *args, **kwargs) -> None:
```

Comment:
Have removed this despite it being introduced in #102, as it's a 3 line function with too much overlap with the constructor. Only downside is now `NumericalHessianCalculator` is public. Is this okay(?!)

Reply:
I don't see a problem with this, I think this is a good move